mflowgen/common/rtl/gen_rtl.sh

```diff
@@ -14,7 +14,7 @@ else
     rm -f $GARNET_HOME/garnet.v
 
     # Build up the flags we want to pass to python garnet.v
-    flags="--width $array_width --height $array_height --pipeline_config_interval $pipeline_config_interval -v --no-sram-stub"
+    flags="--width $array_width --height $array_height --pipeline_config_interval $pipeline_config_interval -v --no-sram-stub --add-pond"
```

Comment:
If it's enabled by default, then we need to change it to `--no-pond` flag to turn it off as an option.